server/routes/courtRegister/courtRegisterController.test.ts

```diff
@@ -22,7 +23,7 @@ describe('Court Register controller', () => {
   afterEach(jest.resetAllMocks)
   describe('getPageOfCourts', () => {
     beforeEach(() => {
-      courtRegisterService = new CourtRegisterService(null) as jest.Mocked<CourtRegisterService>
+      courtRegisterService = new CourtRegisterService({} as HmppsAuthClient) as jest.Mocked<CourtRegisterService>
```

Comment:
obviously parameter is never used since these are mocked, so gone for this compiler hack